### PR TITLE
Network Performance Measurement: Make parameters tunable from outside

### PR DESCRIPTION
**Pull request by VivekThrivikraman-est**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
This PR is for tuning the command parameters for TCP and UDP measurements(so that the metrics makes sense in k8s CI/CD clusters). This PR also contains changes
for making the tuning easier in future by controlling it from the workers deployment yaml file.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: util-images/network/netperfbenchmark/pkg/worker/worker.go [hunk 1]
```diff
@@ -92,10 +93,10 @@ var (
 	// -e : enhanced reports, gives more metrics for udp.
 	// -s : run in server mode.
 	// -P numOfClients: handle <numOfClients> number of clients before disconnecting.
-	udpClientArguments = []string{"-c", "{serverPodIP}", "-u", "-f", "K", "-l", "20", "-b", "1M", "-e", "-i", "1", "-t", "{duration}"}
+	udpClientArguments = []string{"-c", "{serverPodIP}", "-u", "-f", "K", "-e", "-i", "1", "-t", "{duration}"}
 	udpServerArguments = []string{"-s", "-f", "K", "-u", "-e", "-i", "{duration}", "-P", "{numberOfClients}"}
 	tcpServerArguments = []string{"-s", "-f", "K", "-i", "{duration}", "-P", "{numberOfClients}"}
-	tcpClientArguments = []string{"-c", "{serverPodIP}", "-f", "K", "-l", "20", "-b", "1M", "-i", "1", "-t", "{duration}"}
+	tcpClientArguments = []string{"-c", "{serverPodIP}", "-f", "K", "-i", "1", "-t", "{duration}"}
```
Comment (marseel) [inline util-images/network/netperfbenchmark/pkg/worker/worker.go:99]: I see that you removed here few flags and added "-b 750M" in yaml config. What do you think about splitting this change into two PRs(or just commits)? One PR containing logic for parsing flags, moving the one that you want to change to yaml file and second PR modifying flags? So first PR will be no-op and second will modify behaviour.

**Review by marseel — COMMENTED** [review]
Also, quick question, do you want me to build and push this image to repository?

File: util-images/network/netperfbenchmark/pkg/worker/worker.go (hunk 1 above)
Comment (VivekThrivikraman-est) [reply util-images/network/netperfbenchmark/pkg/worker/worker.go:99]: Sure,will raise 2 seperate PR's.

Comment (VivekThrivikraman-est) [reply util-images/network/netperfbenchmark/pkg/worker/worker.go:99]: I have created https://github.com/kubernetes/perf-tests/pull/1853/ for yaml changes.

**Comment by marseel** [comment]
/lgtm
/hold
I've build image and pushed it to gcr. Do you want to test it before we merge this change?

**Comment by VivekThrivikraman-est** [comment]
> /lgtm
> /hold
> I've build image and pushed it to gcr. Do you want to test it before we merge this change?

I have tested with the new image from gcr, looks good.Thanks!

**Comment by marseel** [comment]
Ah, I don't have power to approve in this directory ;) 
/assign @wojtek-t

**Pull request merged**